.github/workflows/linting.yml

```
@@ -0,0 +1,39 @@
+name: GitHub Actions workflow Linting
+
+on:
+  workflow_call:
+  push:
+    branches:
+      - "main"
+  pull_request:
+    paths:
+      - '.github/workflows/*.yml'
+  workflow_dispatch:
+
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+      - name: Checkout code
+        with:
+          persist-credentials: false
+        uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install zizmor==0.9.2
+
+      # Apply GitHub Actions linter, zizmor
+      - name: zizmor
+        if: ${{ !cancelled() }}
+        run: |
+          zizmor .github/workflows/*.yml
```

Using the "auditor" persona reports everything that zizmor found (https://woodruffw.github.io/zizmor/usage/#using-personas). The "regular" persona (default) only flags security issues.

```suggestion
          zizmor --persona=auditor .github/workflows/*.yml
```

---

Actually I reverted this in https://github.com/mesh-adaptation/docs/pull/117/commits/8c57fbd9e24b559579c8fb9a98840a1081b854e3 since these low-level findings caused the CI to fail